tests/test_server.py

```diff
@@ -255,7 +278,7 @@ def setUp(self):
         self.reactor = ThreadedMemoryReactorClock()
 
     def test_good_response(self):
-        def callback(request):
+        async def callback(request):
```

Comment:
Somewhat unrelated, but the `TestResource._async_render_GET` awaits this function so it should be async.